packages/ckeditor5-package-generator/lib/templates/ts/package.json

```diff
@@ -70,11 +70,16 @@
     "@types/node": "^17.0.23",
     "@typescript-eslint/eslint-plugin": "^5.18.0",
     "@typescript-eslint/parser": "^5.18.0",
+    "css-loader": "^5.2.7",
     "eslint": "^7.32.0",
     "eslint-config-ckeditor5": ">=<%= packageVersions.eslintConfigCkeditor5 %>",
     "http-server": "^14.1.0",
     "husky": "^4.2.5",
     "lint-staged": "^10.2.6",
+    "postcss-loader": "^4.3.0",
+    "postcss": "^8.4.12",
+    "raw-loader": "^4.0.2",
+    "style-loader": "^2.0.0",
```

Comment:
L73 + L79-L82: These packages should be installed as dependencies of the `@ckeditor/ckeditor5-package-tools` package. It provides building scripts that require these dependencies.

We need to figure out another way to force installing dependencies instead of looking for hoisted ones.

Reply:
Looks like most of the issues can be resolved by using `bundledDependencies`. Bundled dependencies are bundled with the npm package when publishing. 

There is one package that is required by almost all ckeditor5 packages: `css-loader`, so we have to keep it in the template package.json as we don't want to update ~ 20 packages.